AirBench/Controllers/ReviewController.cs

```diff
@@ -41,6 +41,9 @@ public ActionResult Create(CreateReview review, int id)
                 Review newReview = new Review(0, review.Rating, review.Decription);
 
                 Bench bench = benchRepository.GetBenchById(id);
+                bench.Reviews.Add(newReview);
+                bench.CalculateRating(bench.Reviews);
```

Comment:
Interesting approach to calculate and store a bench's average rating in the database. Using this approach can lead to issues though with keeping the average rating properly in sync with the actual average rating based upon the bench's related reviews. Because of this, it's generally best to avoid storing calculated data in the database, instead preferring to calculate the value when it's needed.